monitoring.adoc

```diff
@@ -240,10 +240,10 @@ with `iostat -x`).
 == Configuring Denial of Service Filter Parameters
 
 The denial-of-service filter (DoSFilter) limits exposure to requests
-flooding.  The DoSFilter throttles clients sending a large number of
+flooding over HTTP/HTTPS.  The DoSFilter throttles clients sending a large number of
 requests over a short period of time.
 
-The DoSFilter is enabled by default on {product-abbrev} and is applied to all requests.
+The DoSFilter is enabled by default on {product-abbrev} and is applied to all HTTP and HTTPS requests. DosFilter dis not applied to POP3, IMAP or SMTP requests.
```

Comment:
nit: typo ```dis``` → ```is```
- we could also reference zimbraPasswordLockout* attributes if we want to point them in something else they may be interested in

Reply:
I added a reference to the section that describes password lockout policy. Unfortunately, we do not have a section that describes how to set password lockout policy via CLI and related LDAP attributes and I think adding that section is out of scope of this bug.

Reply:
I'd suggest future proofing this.  e.g. Use "DosFilter is only applied to HTTP and HTTPS requests, in other words, it does **not** affect requests for any other protocols like POP3, IMAP or SMTP."